liquid_node/configuration.py

```diff
@@ -633,6 +648,11 @@ def tag(name):
         if name in ['dokuwiki', 'nextcloud']:
             return tag('liquid-' + name)
 
+        if name == 'bbb':
+            bbb_gl = tag('bbb-gl')
+            bbb_redis = tag('bbb-redis')
+            return f'greenlight: {bbb_gl}, redis: {bbb_redis}'
```

Comment:
let's only display the greenlight version, these are user-facing version numbers, we don't want to have too many of them

Reply:
done